oidc-controller/api/routers/oidc.py

```diff
@@ -123,9 +126,7 @@ async def get_authorize(request: Request, db: Database = Depends(get_db)):
     else:
         wallet_did = client.get_wallet_did(public=use_public_did)
 
-        byo_attachment = PresentProofv10Attachment.build(
-            pres_exch_dict["presentation_request"]
-        )
+        byo_attachment = PresentProofv10Attachment.build(pres_exch_dict["pres_request"])
```

Comment:
This is the difference in the response from created presentation that is being passed to the byo_attachment

![image](https://github.com/user-attachments/assets/fbdaea0a-b36f-41ac-a9a3-c9bc70f9f798)